src/charts/aws-node-termination-handler-2/templates/node.k8s.aws_terminators.yaml

```diff
@@ -0,0 +1,209 @@
+                    default:
+                      {{- toYaml . | nindent 22 }}
+                    {{- end }}
+                  maxNumberOfMessages:
+                    description: |
+                      The maximum number of messages to return. Amazon SQS never returns more messages
+                      than this value (however, fewer messages might be returned). Valid values:
+                      1 to 10.
+                    type: integer
+                    format: int64
+                    {{- with .Values.terminator.defaults.sqs.maxNumberOfMessages }}
+                    default: {{ . }}
+                    {{- end }}
+                  messageAttributeNames:
+                    description: |
+                      The name of the message attribute, where N is the index.
+
+                        * The name can contain alphanumeric characters and the underscore (_),
+                        hyphen (-), and period (.).
+
+                        * The name is case-sensitive and must be unique among all attribute names
+                          for the message.
+
+                        * The name must not start with AWS-reserved prefixes such as AWS. or Amazon.
+                          (or any casing variants).
+
+                        * The name must not start or end with a period (.), and it should not
+                          have periods in succession (..).
+
+                        * The name can be up to 256 characters long.
+
+                        When using ReceiveMessage, you can send a list of attribute names to receive,
+                        or you can return all of the attributes by specifying All or .* in your request.
+                        You can also use all message attributes starting with a prefix, for example
+                        bar.*.
+                    type: array
+                    items:
+                      type: string
+                    {{- with .Values.terminator.defaults.sqs.messageAttributeNames }}
+                    default:
+                      {{- toYaml . | nindent 22 }}
+                    {{- end }}
+                  queueUrl:
+                    description: |
+                      The URL of the Amazon SQS queue from which messages are received.
+
+                        * Queue URLs and names are case-sensitive.
+
+                        * QueueUrl is a required field
+                    type: string
+                  visibilityTimeoutSeconds:
+                    description: |
+                      The duration (in seconds) that the received messages are hidden from subsequent
+                      retrieve requests after being retrieved by a ReceiveMessage request.
+                    type: integer
+                    format: int64
+                    {{- with .Values.terminator.defaults.sqs.visibilityTimeoutSeconds }}
+                    default: {{ . }}
+                    {{- end }}
+                  waitTimeSeconds:
```

Comment:
this also seems unnecessary to expose especially since you need knowledge of the http response timeout configured for aws sdk.

Reply:
I don't follow what you mean.

As I understand the [docs](https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/sqs-visibility-timeout.html), each request may specify a visibility timeout.  If not specified then a default value from the client or the queue's config will be used. NTH v1 hardcodes the value [here](https://github.com/aws/aws-node-termination-handler/blob/33c6238206b28e38062692188082aaba7287aa0b/pkg/monitor/sqsevent/sqs-monitor.go#L232).

Reply:
I was referring to `waitTimeSeconds`

Reply:
Thanks for clarifying.  So the value should just be hardcoded like in v1 ([here](https://github.com/aws/aws-node-termination-handler/blob/33c6238206b28e38062692188082aaba7287aa0b/pkg/monitor/sqsevent/sqs-monitor.go#L233))?  For reference, is there documentation why that value was chosen to work with the aws sdk in NTH v1?

Reply:
it's the max time you can wait on a long poll: https://docs.aws.amazon.com/AWSSimpleQueueService/latest/SQSDeveloperGuide/working-with-messages.html#:~:text=The%20maximum%20long%20polling%20wait%20time%20is%2020%20seconds